auth/src/main/java/org/aerogear/auth/utils/AuthStateManager.java

```diff
@@ -0,0 +1,60 @@
+package org.aerogear.auth.utils;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import org.aerogear.auth.credentials.OIDCCredentials;
+import org.json.JSONException;
+
+/**
+ * Saves, retrieves and delete a token.
+ */
+public class AuthStateManager {
+
+    private static final String STORE_NAME = "AuthState";
```

Comment:
we might want to namespace the store name here to avoid conflicts. One option could be the package name